src/flash.test.ts

```diff
@@ -0,0 +1,221 @@
+import tap from 'tap'
+import Fastify from 'fastify'
+import fastifyCookie from 'fastify-cookie'
+import fastifySession from 'fastify-session'
+
+import fastifyFlash from '.'
+import { ExtendedRequest, ExtendedReply } from './flash'
+
+tap
+  .test('should set error message and and clear up after displaying.', async t => {
```

Comment:
Can you rewrite this to  `test('should set error message and and clear up after displaying.')`, using destructuring on import